numpy/random/mtrand/mtrand.pyx

```diff
@@ -1732,17 +1749,17 @@ cdef class RandomState:
         """
         if high is None:
             warnings.warn(("This function is deprecated. Please call "
-                           "randint(1, {low} + 1) instead".format(low=low)),
+                           "randint(low=1, high={low} + 1) instead".format(low=low)),
```

Comment:
I disagree : the inclusion of `low=` and `high=` not only reflects the changed API but also encourages what IMO is good practice in clearly specifying keyword arguments, especially since there are no more positional arguments.